api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -911,4 +911,39 @@ public PersonAddress build() {
 			return personAddress;
 		}
 	}
+
+	/**
+	 * @see Person#Person(Person)
+	 * @verifies deep copy all collections
+	 */
+	@Test
+	public void Person_shouldDeepCopyAllCollections() throws Exception {
```

Comment:
Yes, I did. Even if I run it again, it does not change anything in those files. Do I miss some settings?